src/main/resources/archetype-resources/deployment/bin/build

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+REPO=074509403805.dkr.ecr.eu-west-1.amazonaws.com
+
+DIR=$( dirname $0 )
+
+cd ${DIR}/../..
+ROOT=`pwd`
+cd -
+
+DOCKER_PATH=${ROOT}/deployment/docker
+
+docker run --rm=true \
+  --user=$(id -u $USER):$(id -g $USER) \
```

Comment:
You can remove `$USER` here. The Jenkins slaves don't set it and it uses the current user, anyway.